src/components/Header/HeaderUserMenu/HeaderUserMenu.test.js

```diff
@@ -71,4 +77,26 @@ describe('Header user menu', () => {
     );
     expect(getByText('Upgrade')).toBeInTheDocument();
   });
+
+  it('user have SMS enabled', () => {
+    const { getByText } = render(
```

Comment:
Can add the comments like `// Arrange // Act // Assert` ?

Reply:
If I'm not wrong, I didn't see them in any test file. Maybe this could be part of a refactor.

Reply:
When we remember we add this comments, you are more than welcome to remind us about this in the future too 😝